models/workday_history/workday__monthly_summary.sql

```diff
@@ -0,0 +1,112 @@
+{{ config(enabled=var('employee_history_enabled', False)) }}
+
+with row_month_partition as (
+
+    select *, 
+        {{ dbt.date_trunc("month", "date_day") }} as date_month,
+        row_number() over (partition by employee_id, extract(year from date_day), extract(month from date_day) order by date_day desc) AS recent_dom_row
+    from {{ ref('workday__employee_daily_history') }}
+    order by employee_id, date_day
```

Comment:
Good call. This was mostly for development/validation purposes (since otherwise everything was very jumbled), but isn't necessary in the final model.